### DISCUSSION TITLE
mobile view is not responsive enough

### DISCUSSION THREAD

**Discussion by sinkaroid** [apidoc/apidoc#1320] [Q&A]
Thanks for the low effort documentation with only js or ts definitions

Had some issues here: on mobile, codeblocks, sidenav is not slicing properly and some sentence was cutting off
I'm on `apidoc@latest` and without any custom template here
![Screenshot_110](https://user-images.githubusercontent.com/12372481/209268583-9f625382-39d3-44ac-8e18-b2707842f037.png)

**Comment by NicolasCARPi** [comment]
Hello,

You are welcome to contribute.

Best,
~Nico
